vignettes/datatable-reshape.Rmd

```diff
@@ -157,7 +157,7 @@ DT.m1[, c("variable", "child") := tstrsplit(variable, "_", fixed = TRUE)]
 DT.c1 = dcast(DT.m1, family_id + age_mother + child ~ variable, value.var = "value")
 DT.c1
 
-str(DT.c1) ## gender column is character type now!
+str(DT.c1) ## gender column is character IDate now!
```

Comment:
```suggestion
str(DT.c1) ## gender column is class IDate now!
```

Reply:
Actually this is an interesting point, this example was designed for when `dob_child1` etc. would be read by `fread()` as type `character`, but now they're recognized as dates and read as `IDate`.

Before this example illustrated `gender` being cast from `integer` to `character`, now it only goes `integer`->`integer`, but gains the class attribute. Maybe we should revisit this.